.env.example

```
@@ -76,3 +75,5 @@ RETENTION_CRON="* * * * *"
 
 TRUSTED_DOMAINS="http://localhost:8887, http://localhost:3055, http://localhost:3056"
 
+#OPENAPI_ENABLED=true
+#API_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)